### PR TITLE
Use Statefulset instead in e2e and controller

### PR DESCRIPTION
**Pull request by resouer**
Quick fix ref: #35534

We should finish the issue to meet v1.6 milestone.

### CODE REVIEW

**Comment by resouer** [comment]
kindly ping @mikedanese, could you please give this a quick review? This is easy to become need rebase.

File: cmd/kube-controller-manager/app/apps.go [hunk 1]
```diff
@@ -22,15 +22,15 @@ package app
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	petset "k8s.io/kubernetes/pkg/controller/petset"
+	statefulset "k8s.io/kubernetes/pkg/controller/statefulset"
```
Comment (mikedanese) [inline cmd/kube-controller-manager/app/apps.go]: Is there a reason this import is named?

Comment (resouer) [inline cmd/kube-controller-manager/app/apps.go]: Emm ... seems not

**Comment by resouer** [comment]
@k8s-bot kops aws e2e test this

**Pull request merged**